pill-press-app/ClientApp/src/app/dashboard-lite/dashboard-lite.component.ts

```diff
@@ -54,7 +54,7 @@ export class DashboardLiteComponent implements OnInit {
 
   }
 
-  startNewLicenceApplication() {
+  startNewWaiverApplication() {
```

Comment:
I see these methods are not the sending over the application type. Is this intentional?